File: RepoTransBench/target_projects/Java/C#/vatri_spring-rest-ecommerce/public_tests/StoragePropertiesPublicTest.cs
using Xunit;
using Vatri.Ecommerce.Storage;

namespace PublicTests
{
    public class StoragePropertiesPublicTest
    {
        [Fact]
        public void TestDefaultLocationPublic()
        {
            var properties = new StorageProperties();
            Assert.NotEqual("somewhereelse", properties.GetLocation());
            Assert.Equal("uploads", properties.GetLocation());
        }
        [Fact]
        public void TestSetLocationPublic()
        {
            var properties = new StorageProperties();
            properties.SetLocation("my_new_location");
            Assert.Equal("my_new_location", properties.GetLocation());
            Assert.NotEqual("abc", properties.GetLocation());
        }
    }
}

File: RepoTransBench/source_projects/Java/vatri_spring-rest-ecommerce/src/test/java/net/vatri/ecommerce/storage/StoragePropertiesPublicTest.java
package net.vatri.ecommerce.storage;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class StoragePropertiesPublicTest {
    @Test
    void testDefaultLocationPublic() {
        StorageProperties properties = new StorageProperties();
        // Use assertNotEquals with old value and set to a different thing after to check
        assertNotEquals("somewhereelse", properties.getLocation());
        assertEquals("uploads", properties.getLocation());
    }

    @Test
    void testSetLocationPublic() {
        StorageProperties properties = new StorageProperties();
        properties.setLocation("my_new_location");
        assertEquals("my_new_location", properties.getLocation());
        assertNotEquals("abc", properties.getLocation());
    }
}